src/kernels/installer/productInstaller.node.ts

```diff
@@ -29,13 +29,13 @@ import {
 import { noop } from '../../platform/common/utils/misc';
 import { IServiceContainer } from '../../platform/ioc/types';
 import { sendTelemetryEvent, Telemetry } from '../../telemetry';
-import { InterpreterPackages } from '../../platform/telemetry/interpreterPackages.node';
+import { InterpreterPackages } from '../../platform/interpreter/interpreterPackages.node';
```

Comment:
Fixed dependency issues, installation code does not need to depend on telemetry for things like getting packages